install/kubernetes/cilium/values.yaml

```diff
@@ -420,26 +420,47 @@ encryption:
   # -- Encryption method. Can be either ipsec or wireguard.
   type: ipsec
 
-  # -- Name of the key file inside the Kubernetes secret configured via secretName.
+  # -- Deprecated in favor of encryption.ipsec.keyFile.
+  # Name of the key file inside the Kubernetes secret configured via secretName.
   # This option is only effective when encryption.type is set to ipsec.
   keyFile: keys
 
-  # -- Path to mount the secret inside the Cilium pod.
+  # -- Deprecated in favor of encryption.ipsec.mountPath.
+  # Path to mount the secret inside the Cilium pod.
   # This option is only effective when encryption.type is set to ipsec.
   mountPath: /etc/ipsec
 
-  # -- Name of the Kubernetes secret containing the encryption keys.
+  # -- Deprecated in favor of encryption.ipsec.secretName.
+  # Name of the Kubernetes secret containing the encryption keys.
   # This option is only effective when encryption.type is set to ipsec.
   secretName: cilium-ipsec-keys
 
   # -- Enable encryption for pure node to node traffic.
   # This option is only effective when encryption.type is set to ipsec.
   nodeEncryption: false
 
-  # -- The interface to use for encrypted traffic.
+  # -- Deprecated in favor of encryption.ipsec.interface.
+  # The interface to use for encrypted traffic.
   # This option is only effective when encryption.type is set to ipsec.
   # interface: eth0
 
+  ipsec:
+    # -- Name of the key file inside the Kubernetes secret configured via secretName.
+    # This option is only effective when encryption.type is set to ipsec.
```

Comment:
I wonder if these warnings are still necessary now that the options are attached to `encryption.ipsec`?

Reply:
Yeah, I don't think we need them anymore :+1: